Config/core_packs_platform_list.json

```diff
@@ -1,16 +1,289 @@
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS",
+                "identity_threat"
+            ]
+        },
+        {
+            "id": "CommonPlaybooks",
+            "supportedModules": [
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS"
+            ]
+        },
+        {
+            "id": "CommonScripts",
+            "supportedModules": [
+                "C1",
+                "C3",
+                "X0",
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS",
+                "identity_threat"
+            ]
+        },
+        {
+            "id": "CommonTypes",
+            "supportedModules": [
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS"
+            ]
+        },
+        {
+            "id": "Core",
+            "supportedModules": [
+                "C3",
+                "X0",
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS"
+            ]
+        },
+        {
+            "id": "CoreAlertFields",
+            "supportedModules": [
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS"
+            ]
+        },
+        {
+            "id": "CortexAttackSurfaceManagement",
+            "supportedModules": [
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS"
+            ]
+        },
+        {
+            "id": "DemistoLocking",
+            "supportedModules": [
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS"
+            ]
+        },
+        {
+            "id": "DemistoRESTAPI",
+            "supportedModules": [
+                "C1",
+                "C3",
+                "X0",
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS",
+                "identity_threat"
+            ]
+        },
+        {
+            "id": "EDL",
+            "supportedModules": [
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS"
+            ]
+        },
+        {
+            "id": "FeedMitreAttackv2",
+            "supportedModules": [
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS"
+            ]
+        },
+        {
+            "id": "FeedUnit42v2",
+            "supportedModules": [
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS"
+            ]
+        },
+        {
+            "id": "FiltersAndTransformers",
+            "supportedModules": [
+                "C1",
+                "C3",
+                "X0",
+                "X1",
+                "X3",
+                "X5",
+                "ENT_PLUS",
+                "identity_threat"
```

Comment:
What is identity_threat?

Reply:
The code for `CIEM` (got it from @GuyAfik )

Reply:
@ilappe that's not true,
`identity_threat = ITDR`